draft-roadmap-2024.md

```diff
@@ -1,49 +1,81 @@
 The roadmap is one of the important artifacts on the website.
 
-The draft roadmap 2024 was created based on information from the Attention Ubiquity Core Team https://github.com/ubiquity
+The draft roadmap was created based on information from the Attention Ubiquity Core Team https://github.com/ubiquity and Alex's recommendations https://github.com/ubiquity/research/pull/130#discussion_r1485777642
+
 Once the content and structure of the roadmap are approved, a visually appealing roadmap will be created.
 
-The draft of the roadmap 2024 content and structure for discussion:
+The draft of the roadmap Q1 2024 - Q2 2025 for discussion:
 
 ## Product Development
 
-### Q1
+### Q1 2024
+
+Security assessment
+
+### Q2 2024
+
+Ubiquity Dollar stable version launch
+
+Ubiquity Card integration to DAO OS
+
+### Q3 2024
+
+DAO OS Kernel stable version launch
+
+Ubiquity Dollar integration to DAO OS
+
+### Q4 2024
+
+DAO OS Plugins open for contribution
+
+DAO OS AI agents alpha version 
```

Comment:
```suggestion

```

Replaced in favor of plugins. Technically it would just be a class of plugins. For example, we create an agent template (not a priority now) and then let the user configure the initialization prompt in plain english to configure it, for example.